server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -43,12 +43,15 @@ const AssessmentDayDataController = {
         query
       )
       console.log('PARTICIPANT DATA')
-      console.dir({ participantAssessmentData }, { depth: null })
+      console.dir(
+        { participantAssessmentData, metadata, assessment_variables },
+        { depth: null }
+      )
       let sortedDayData = participant_assessments
       let maxDayInDayData = Math.max(
         ...participant_assessments.map((pa) => pa.day)
       )
-
+      console.dir(participant_assessments, { depth: null })
```

Comment:
should we also log the parsed consent?

Reply:
does not hurt[d7baa42](https://github.com/AMP-SCZ/dpdash/pull/750/commits/d7baa42e96ab2cb225cded3229179c855ba3bcc2)